src/extractors/custom/www.moongift.jp/index.js

```diff
@@ -0,0 +1,27 @@
+export const WwwMoongiftJpExtractor = {
+  domain: 'www.moongift.jp',
+
+  title: {
+    selectors: ['h1.title a'],
+  },
+
+  author: null,
+
+  date_published: null,
```

Comment:
This could actually be obtained using something like `ul.meta li:not(.social):first-of-type`

Reply:
Thank you! I fixed.